core/loader.ts

```diff
@@ -15,14 +15,19 @@ import RoundService from 'core/service/round';
 import BlockService from 'core/service/block';
 import RoundRepository from 'core/repository/round';
 
+// @ts-ignore
+BigInt.prototype.toJSON = function () {
```

Comment:
Why here? Why do we need this? Just for logging?

Reply:
It is serializer